RabbitMQ.Stream.Client/AMQP/FormatCode.cs

```diff
@@ -0,0 +1,98 @@
+namespace RabbitMQ.Stream.Client.AMQP
+{
+    public static class FormatCode
```

Comment:
Would it be possible to actually merge this constants into above class, `DescribedFormatCode`? I mean, rather than having two classes.